### PR TITLE
Documentation for Azure Files and EFS

### PR DESCRIPTION
**Pull request by atsikham**
Task #1974. It already has been supported but not documented.

### CODE REVIEW

**Comment by seriva** [comment]
This needs changes for sure because of #2645.

**Comment by seriva** [comment]
First we need to merge #2643.

File: data/aws/defaults/infrastructure/efs-storage.yml [hunk 1]
```diff
@@ -12,6 +12,3 @@ specification:
   mount_targets: [] #TO_BET_SET
 #    - name: mount-name
 #      subnet_name: source_subnet_name
-  security:
-    populate_sg_rules: true # when set to true, security group for EFS will allow traffic from other clusters, groups
```
Comment (atsikham) [inline data/aws/defaults/infrastructure/efs-storage.yml:16]: Haven't found places where it's used.

File: docs/home/howto/PERSISTENT_STORAGE.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    path: /
+    enable: true
+    capacity: 50
+```
+
+#### Additional configuration
+
+If `provider: aws` is specified, EFS storage is always created and can be used with persistent volumes created by the 
+user. It is possible to create a separate EFS and use it. For more information check Kubernetes 
+[NFS](https://kubernetes.io/docs/concepts/storage/volumes/#nfs) storage documentation. There is another way exists
```
Comment (plirglo) [inline docs/home/howto/PERSISTENT_STORAGE.md]:
```suggestion
[NFS](https://kubernetes.io/docs/concepts/storage/volumes/#nfs) storage documentation. There is another way
```

**Review by plirglo — CHANGES_REQUESTED** [review]

**Review by plirglo — APPROVED** [review]

**Review by seriva — APPROVED** [review]

**Pull request merged**